arch/arm/soc/st_stm32/stm32f4/dts.fixup

```diff
@@ -83,4 +83,12 @@
 #define FLASH_DEV_BASE_ADDRESS		        ST_STM32F4_FLASH_CONTROLLER_40023C00_BASE_ADDRESS_0
 #define FLASH_DEV_NAME			        ST_STM32F4_FLASH_CONTROLLER_40023C00_LABEL
 
+#define CONFIG_USB_BASE_ADDRESS			ST_STM32_OTGFS_50000000_BASE_ADDRESS
+#define CONFIG_USB_IRQ				ST_STM32_OTGFS_50000000_IRQ_OTGFS
+#define CONFIG_USB_IRQ_PRI			ST_STM32_OTGFS_50000000_IRQ_OTGFS_PRIORITY
```

Comment:
Where is the OTGFS_PRIORITY defined in the stm32f4.dtsi file?

Reply:
Never mind, I got it now. It's defined by the DTC from the second parameter in the interrupts property.

Reply:
IRQ_OTGFS_PRIORITY is the second number in interrupts property.
interrupts = <67 **0**>;